packages/twenty-front/src/modules/auth/services/AuthService.ts

```diff
@@ -57,6 +57,7 @@ const renewTokenMutation = async (
 /**
  * Renew token and update cookie storage
  * @param uri string | UriFunction | undefined
+ * @param tokenPair AuthTokenPair | undefined | null
  * @returns TokenPair
  */
 export const renewToken = async (
```

Comment:
about this doc, we don't document the service method in the codebase for now. The general assumption is that the parameter naming should be clear enough and the types are already present in the signature.
What is the motivation behind the doc here?

If we decide to document methods (why not if there is a good reason), we should apply this to all methods the consistency within the project is very important!

Reply:
Just had a warning in my ide for that missing parameter doc

Reply:
Yep but it's because there is a parameters doc :p we could also completely remove it